tests/test_particle_sets.py

```diff
@@ -291,3 +291,23 @@ def SetLat(particle, fieldset, time, dt):
     assert np.allclose([p.lon for p in pset], lon)
     assert np.allclose([p.lat for p in pset], .6)
     assert np.allclose([p.time for p in pset], 0)
+
+
+@pytest.mark.parametrize('mode', ['scipy', 'jit'])
+@pytest.mark.parametrize(('npart','interval'),[(1,0.2),(5,0.2),(1,1.0),(5,1.0)])
```

Comment:
If you want all combinations of `nart` and `interval`, simply put them in different `@pytest.mark.parametrize` statements?